src/main/java/hudson/plugins/copyartifact/CopyArtifact.java

```diff
@@ -512,10 +533,16 @@ public FormValidation doCheckProjectName(
                 }
             else if (value.indexOf('$') >= 0)
                 result = FormValidation.warning(Messages.CopyArtifact_ParameterizedName());
-            else
+            else {
+                Job<?,?> nearest = Items.findNearest(Job.class, value, anc.getParent());
+                if (nearest != null) {
                 result = FormValidation.error(
                     hudson.tasks.Messages.BuildTrigger_NoSuchProject(
-                        value, Items.findNearest(Job.class, value, anc.getParent()).getName()));
+                        value, nearest.getName()));
+                } else {
+                    result = FormValidation.error(hudson.tasks.Messages.BuildTrigger_NoProjectSpecified());
+                }
+            }
```

Comment:
```
    <BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE'
        priority='Normal'
        category='STYLE'
        message='Possible null pointer dereference in hudson.plugins.copyartifact.CopyArtifact$DescriptorImpl.doCheckProjectName(Job, String) due to return value of called method'
        lineNumber='516'/>
```
This is a issue pointed in #61, but that pull request was replaced with other pull requests and this was not fixed.